src/app/components/cesium/map/map.js

```diff
@@ -56,6 +55,8 @@ class CesiumComponent extends Component {
 
     if (zoomLevel) this.handleZoom(zoomLevel)
     if (lockNavigation) return disablePanning(viewer)
+
+    this.state.viewer = viewer
```

Comment:
Why do you mutate the state?
Besides, I don't think it's needed cause after the call to mountMap you use the resulting viewer to setState inside bindMap.